src/RegistrationController.js

```diff
@@ -148,6 +150,16 @@ export default BaseLoginController.extend({
     this.$el.find('.okta-form-infobox-error p').text(errorSummary);
   },
 
+  renderLegacyLocationErrorIfNeeded: function(location, errorSummary) {
+    // replace generic error message with errorSummary for v1 SIW
+    // this makes sure that with legacy location that starts with `data.userProfile`
+    // we still see the errorSummary in the error banner instead of only a generic error
+    // See example in https://developer.okta.com/docs/reference/registration-hook/#sample-json-payload-of-request
+    if (location && location.startsWith('data.userProfile')) {
```

Comment:
`startsWith` is [not supported on ie11](https://caniuse.com/?search=startsWith), a browser version we support, and I'm not sure if SIW's babel version polyfills it. You can try using another method or test ie11 using the Sauce Labs chiclet.